plugins/woocommerce/patterns/page-coming-soon-split-right-image.php

```diff
@@ -0,0 +1,65 @@
+						<!-- /wp:image --></div>
+					<!-- /wp:group -->
+
+					<!-- wp:paragraph {"align":"center","style":{"typography":{"fontSize":"13px","lineHeight":"1.6","letterSpacing":"-0.13px","fontStyle":"normal","fontWeight":"400"},"color":{"text":"#2f2f2f"},"elements":{"link":{"color":{"text":"#2f2f2f"}}},"spacing":{"margin":{"top":"0px","bottom":"0px"}}}} -->
+					<p class="has-text-align-center has-text-color has-link-color" style="color:#2f2f2f;margin-top:0px;margin-bottom:0px;font-size:13px;font-style:normal;font-weight:400;letter-spacing:-0.13px;line-height:1.6"><?php echo esc_html_x( 'Dedicated to providing top-quality bikes, accessories, and expert advice for riders of all experience levels. Stay tuned.', 'Used in the paragraph of the coming soon page', 'woocommerce' ); ?></p>
+					<!-- /wp:paragraph -->
+
+					<!-- wp:group {"style":{"spacing":{"padding":{"top":"0px","bottom":"0px"},"margin":{"top":"24px","bottom":"32px"},"blockGap":"0"},"layout":{"selfStretch":"fit","flexSize":null}},"layout":{"type":"constrained","justifyContent":"center"}} -->
+					<div class="wp-block-group" style="margin-top:24px;margin-bottom:32px;padding-top:0px;padding-bottom:0px"><!-- wp:template-part {"slug":"coming-soon-social-links","theme":"woocommerce/woocommerce","tagName":"div","align":"center","className":"is-size-fit-content"} /--></div>
+					<!-- /wp:group --></div>
+				<!-- /wp:group --></div>
+			<!-- /wp:column -->
+
+			<!-- wp:column {"verticalAlignment":"stretch","layout":{"type":"default"}} -->
+			<div class="wp-block-column is-vertically-aligned-stretch"><!-- wp:cover {"url":"<?php echo esc_url( $right_image ); ?>","id":34,"dimRatio":0,"customOverlayColor":"#cd7550","isUserOverlayColor":false,"focalPoint":{"x":0.5,"y":0.5},"minHeight":100,"minHeightUnit":"%","isDark":false,"className":"woocommerce-split-right-image-cover","style":{"spacing":{"padding":{"top":"126px","bottom":"126px","left":"76px","right":"75px"}}},"layout":{"type":"default"}} -->
+				<div class="wp-block-cover is-light woocommerce-split-right-image-cover" style="padding-top:126px;padding-right:75px;padding-bottom:126px;padding-left:76px;min-height:100%"><span aria-hidden="true" class="wp-block-cover__background has-background-dim-0 has-background-dim" style="background-color:#cd7550"></span><img class="wp-block-cover__image-background wp-image-34" alt="" src="<?php echo esc_url( $right_image ); ?>" style="object-position:50% 50%" data-object-fit="cover" data-object-position="50% 50%"/><div class="wp-block-cover__inner-container"><!-- wp:paragraph {"align":"center","placeholder":"Write title…","style":{"typography":{"fontSize":"75px","fontStyle":"normal","fontWeight":"400","lineHeight":"1.3"},"color":{"text":"#ffffff80"},"elements":{"link":{"color":{"text":"#ffffff80"}}}},"fontFamily":"<?php echo esc_attr( $cardo_font_family ); ?>"} -->
```

Comment:
I find this to be better, especially on smaller screens where the bike will be in focus

<img width="493" alt="image" src="https://github.com/user-attachments/assets/b3ae6afe-468e-4d0a-9bcb-dbbe71c40b20" />

```suggestion
			<div class="wp-block-column is-vertically-aligned-stretch"><!-- wp:cover {"url":"<?php echo esc_url( $right_image ); ?>","id":34,"dimRatio":0,"customOverlayColor":"#cd7550","isUserOverlayColor":false,"focalPoint":{"x":0.5,"y":0.8},"minHeight":100,"minHeightUnit":"%","isDark":false,"className":"woocommerce-split-right-image-cover","style":{"spacing":{"padding":{"top":"126px","bottom":"126px","left":"76px","right":"75px"}}},"layout":{"type":"default"}} -->
				<div class="wp-block-cover is-light woocommerce-split-right-image-cover" style="padding-top:126px;padding-right:75px;padding-bottom:126px;padding-left:76px;min-height:100%"><span aria-hidden="true" class="wp-block-cover__background has-background-dim-0 has-background-dim" style="background-color:#cd7550"></span><img class="wp-block-cover__image-background wp-image-34" alt="" src="<?php echo esc_url( $right_image ); ?>" style="object-position:50% 80%" data-object-fit="cover" data-object-position="50% 80%"/><div class="wp-block-cover__inner-container"><!-- wp:paragraph {"align":"center","placeholder":"Write title…","style":{"typography":{"fontSize":"75px","fontStyle":"normal","fontWeight":"400","lineHeight":"1.3"},"color":{"text":"#ffffff80"},"elements":{"link":{"color":{"text":"#ffffff80"}}}},"fontFamily":"<?php echo esc_attr( $cardo_font_family ); ?>"} -->
```